python/tidy/servo_tidy/tidy.py

```diff
@@ -19,6 +19,25 @@
 import sys
 from licenseck import MPL, APACHE, COPYRIGHT, licenses_toml, licenses_dep_toml
 import colorama
+import toml
+
+# Load configs from servo-tidy.toml if config file exists
+if os.path.exists("./servo-tidy.toml"):
```

Comment:
Please catch the case in which the config file does not exist, as well. Printing "./servo-tidy.toml config file is required but was not found" and exiting would be fine.